en/docs/monitoring-and-insights/observability-overview.md

```diff
@@ -56,25 +56,38 @@ By default, the time range selected for the **Throughput & Latency** graphs is t
 A **bin** is a horizontal section of the graph for a particular period. A bin consists of the following items:
 
 - **Date/Time:** The timestamp for the log entries as they started to appear.
-- **Logs:**  List of log entries and their respective log count that occurred within the bins time frame. These log entries appear according to precedence. The error logs are listed first, followed by the info logs. Each bin shows a maximum of five log entries sorted by the log count.
+- **Logs:**  List of log entries and their respective log count that occurred within the bin's time frame. These log entries appear according to precedence. The error logs are listed first, followed by the info logs. Each bin shows a maximum of five log entries sorted by the log count.
 - **Error:** The number of HTTP errors that occurred at the selected time.
 - **TP:** Throughput of the requests at the selected time (req/s).
 - **Latency:** Latency of the request at the selected time (ms).
 - **CPU:** CPU usage at the selected time (millicores).
 - **Memory:** Memory usage at the selected time (MiB).
 
+## Logs
 
-## View and filter all logs
+The Choreo log panel offers a centralized view to access comprehensive runtime logs of the components you have deployed on Choreo, enhancing troubleshooting and analysis.
```

Comment:
Shall we say `pane` instead of `panel` to follow the style guide recommendation?

Reply:
done